src/test/datascience/errorHandler.unit.test.ts

```diff
@@ -330,7 +325,7 @@ ImportError: No module named 'xyz'
 
             const expectedMessage = DataScience.failedToStartKernelDueToImportFailure().format('xyz');
 
-            verifyErrorMessage(expectedMessage, 'https://aka.ms/kernelFailuresModuleImportErr');
+            verifyErrorMessage(expectedMessage, 'https://aka.ms/kernelFailuresModuleImportErrFromFile');
```

Comment:
This is the wrong link, the test is now wrong.
If importing `xyz` fails, then this means a module named `xyz` is missing.